jep/1/README.adoc

```diff
@@ -758,6 +851,10 @@ The Python process uses "Rationale" as the heading for the section for describin
 However the meaning of "Rationale" is similar to "Motivation" in some contexts.
 We decided to use "Reasoning" instead to avoid confusion.
 
+Some non-native english speakers commented on the
```

Comment:
Not sure it is really required to keep it in the body of the document.
Maybe we could create some Wiki pages for extra clarification of the top of the JEP document (and extra examples).

Reply:
@oleg-nenashev
I lean towards  including at least a highlight of notable comments about a choice. I feel this kind of comment is exactly what this section is for - part of the "documenting differing opinions".  

Other non-native english speakers will appreciate that we noted this issue.  If someone comes along later with a strong  opinion about this, perhaps considering looking for better options, they will be able to include this in their considerations.

Also, it is a small enough comment that pushing it out of the document body would mean it would almost surely be overlooked.